NamespaceFixer/NamespaceBuilder/XamlNamespaceBuilderService.cs

```diff
@@ -0,0 +1,164 @@
+﻿using System;
+using System.IO;
+using System.IO.Extensions;
+using System.Linq;
+using System.Text.RegularExpressions;
+using System.Xml;
+
+namespace NamespaceFixer.NamespaceBuilder
+{
+    internal class XamlNamespaceBuilderService : INamespaceBuilder
+    {
+        public XamlNamespaceBuilderService(INamespaceAdjusterOptions options)
+        {
+            _options = options;
+        }
+
+        protected Match FindNamespaceMatch(string fileContent)
+        {
+            return Regex.Match(fileContent, @"x:Class=""([a-zA-Z0-9_\.]+)\.[a-zA-Z0-9_]+""");
```

Comment:
This finds the class name not the namespace. The project namespace is under `xmlns:local`